plat/nvidia/tegra/soc/t210/plat_setup.c

```diff
@@ -137,10 +146,76 @@ void plat_early_platform_setup(void)
 	}
 }
 
+/* Secure IRQs for Tegra186 */
+static const interrupt_prop_t tegra210_interrupt_props[] = {
+	INTR_PROP_DESC(TEGRA210_WDT_CPU_LEGACY_FIQ, GIC_HIGHEST_SEC_PRIORITY,
+			GICV2_INTR_GROUP0, GIC_INTR_CFG_EDGE),
+};
+
+void plat_late_platform_setup(void)
```

Comment:
I couldn't find where this function is invoked. Suggest the API name to be made specific to Tegra `plat_tegra_late_platform_setup()`.

Reply:
@soby-mathew 

It will be used in subsequent patches.